easybuild/easyconfigs/a/ARPACK/ARmake-96.inc.patch

```diff
@@ -0,0 +1,56 @@
+--- ARmake.inc.orig	2014-10-15 14:16:00.338607000 +1300
++++ ARmake.inc	2014-10-16 10:29:05.347550000 +1300
+@@ -25,23 +25,23 @@
+ # | other than your home directory.      |
+ # %--------------------------------------%
+ #
+-home = $(HOME)/ARPACK
++home = 
+ #
+ #  %--------------------------------------%
+ #  | The platform identifier to suffix to |
+ #  | the end of library names             |
+ #  %--------------------------------------%
+ #
+-PLAT = SUN4
++PLAT = linux
+ #
+ #  %------------------------------------------------------%
+ #  | The directories to find the various pieces of ARPACK |
+ #  %------------------------------------------------------%
+ #
+-BLASdir      = $(home)/BLAS
+-LAPACKdir    = $(home)/LAPACK
+-UTILdir      = $(home)/UTIL
+-SRCdir       = $(home)/SRC
++BLASdir      = BLAS
++LAPACKdir    = LAPACK
++UTILdir      = UTIL
++SRCdir       = SRC
+ #
+ DIRS        = $(BLASdir) $(LAPACKdir) $(UTILdir) $(SRCdir)
+ #
+@@ -57,7 +57,7 @@
+ # | The name of the libraries to be created/linked to |
+ # %---------------------------------------------------%
+ #
+-ARPACKLIB  = $(home)/libarpack_$(PLAT).a
++ARPACKLIB  = /tmp/libarpack_$(PLAT).a
```

Comment:
OK, then someone will have to look into adding support for new constants like `$EASYBUILD_BUILDDIR` and/or `$EASYBUILD_INSTALLDIR`

maybe @wpoely86 is up for that?